gandalf/build.gradle

```diff
@@ -65,9 +65,9 @@ gnag {
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
 
-    compile 'com.android.support:appcompat-v7:23.2.0'
+    compile 'com.android.support:appcompat-v7:24.2.1'
```

Comment:
In general it would be preferable to separate PRs that bump dependency versions from PRs that add new features unless the feature _requires_ that newer version :)